### DISCUSSION TITLE
Question on getting xdrip to send data to watchface over internet (LTE).

### DISCUSSION THREAD

**Discussion by Npavkov** [NightscoutFoundation/xDrip#3310] [Q&A]
Is there a way to make xdrip send data to watchface over the internet (LTE)?

**Comment by Navid200** [comment]
Do you know a low-cost watch that can access the internet and has a battery that while doing so runs on one charge for 6 hours?

**Comment by Npavkov** [comment]
I am using a Galaxy watch 4

**Reply by Navid200** [reply]
Galaxy watch 4 is WearOS:  https://www.gsmarena.com/samsung_galaxy_watch4-11045.php

Therefore, it cannot access the internet without a phone.

**Comment by Npavkov** [comment]
I would like to be able to have my phone nearby, within 300 ft, and have my watch receive data from xdrip via lte internet. And with LTE and a stand alone data line on the watch it should be able to access internet without the phone at all?

**Reply by Navid200** [reply]
I see that it has WLAN.  
I will ask.

**Reply by Navid200** [reply]
You can set up xDrip to upload to Nightscout.
Then, on the watch, that can access the internet, you can use a browser to access your Nightscout and see your readings and get alerts etc.

**Comment by skritland** [comment]
I have Galaxy Watch 5 with LTE and it is working with no problems. I have xDrip (carelink follower) on my phone, and xDrip wearos application on the watch. Even if phone is far away from the watch and the watch switches from Bluetooth to LTE, it is working with no problems. XDrip is using Google Play services for comminication so it automatically handles bluetooth, wifi and LTE. The only caveat is that phone needs to have the internet connection.
